ziti-tunnel/docker/BUILD.md

```diff
@@ -15,12 +15,11 @@ ziti binaries are in a different artifactory server and/or repository.
 You can also build for other architectures (e.g. linux/arm) with docker's `buildx`
 sub-command. See https://docs.docker.com/buildx/working-with-buildx/ for specifics.
 
-    $ ziti_version="0.5.7-2546"
+    $ ziti_version="0.5.8-2554"
     $ docker buildx create --use
     $ docker buildx build \
         --platform linux/amd64,linux/arm/v7 \
         --build-arg ZITI_VERSION="${ziti_version}" \
-        --build-arg ARTIFACTORY_USERNAME=shawn.carey \
```

Comment:
I think we should keep this line, even if the specific username is ellipsed out. The ARTIFACTORY_USERNAME argument is required, and referring to it here makes this text more usable as a copy/paste source.

Reply:
disregard :)